advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx

```diff
@@ -87,24 +87,29 @@ Here's a sample configuration file in YAML format with Azure specified as the pr
 haStandbyReplicas: 2                                                  # <number:  Number of standby replicas. Field must be specified if user has selected Primary/Standby High Availability cluster type. Default value is 2, valid values [1, 2].>
-provider: azure                                                       # <string:  cloud provider id, valid values ["azure", "aws", "gcp"]>
-cspSubscriptionIDs:                                                   # <list:  cloud provider subscription/account ID, required when cluster is EDB Hosted>
+provider: azure                                                       # <string:  cloud provider id, valid values - Deployment: Your Cloud Account ["azure", "aws", "gcp"], Deployment: BigAnimal ["bah:azure", "bah:aws", "bah:gcp"]>
+cspSubscriptionIDs:                                                   # <list:  cloud provider subscription/account ID, required when cluster is BigAnimal hosted>
   - 123123123                                                         # <string>
   - 456456456                                                         # <string>
-serviceAccountIds:                                                    # <list: A Google Cloud Service Account is used for logs. If you leave this blank, then you will be unable to access log details for this cluster. Required when cluster is EDB Hosted>
+serviceAccountIds:                                                    # <list: A Google Cloud Service Account is used for logs. If you leave this blank, then you will be unable to access log details for this cluster. Required when cluster is BigAnimal hosted>
   - service-account-1234567b@development-data-123456.iam.gserviceaccount.com# <string>
   - service-account-1234567b@development-data-123456.iam.gserviceaccount.com# <string>
 clusterName: biganimal_cluster                                        # <string:  cluster name>
 password: ************                                                # <string:  cluster password (must be at least 12 characters)>
-# refer following link for steps to setup IAM:  https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/01_postgres_access/iam-authentication-for-postgres
+tags:                                                                 # <list: Group clusters and organise across different resource types. This will help to manage resources effectively.>
+  - name: tag1                                                        # <string: assign "tag1" property to the cluster>
+    color: "#c5eae7"                                                  # <string: color for "tag1">
+  - name: tag2                                                        # <string: assign "tag2" property to the cluster>
+    color: "#0b6ff4"                                                  # <string: color for "tag2">
+# refer following link for steps to setup IAM:  https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/01_postgres_access/iam-authentication-for-postgres 
```

Comment:
Wrong link, maybe it should be https://www.enterprisedb.com/docs/edb-postgres-ai/cloud-service/using_cluster/postgres_access/database_authentication/#iam-authentication-for-postgres ?